data/json/items/armor/head_attachments.json

```diff
@@ -693,5 +693,74 @@
     "proportional": { "weight": 0.75, "volume": 1.3, "price": 1.25 },
     "extend": { "flags": [ "UNDERSIZE" ] }
+  },
+  {
+    "id": "outsert_sunlight",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str": "sunlight outSert lens", "str_pl": "sunlight outSert lenses" },
+    "description": "a tinted and flexible plastic outsert intended to be mounted over the eyepiece of a panoramic-view gas mask to protect the wearer from high-level UV light, sudden flashes, and overly bright flares.",
+    "weight": "136 g",
+    "volume": "200 ml",
+    "price": 8999,
+    "price_postapoc": 200,
+    "material": [ "thermo_resin" ],
+    "symbol": "[",
+    "color": "light_gray",
+    "material_thickness": 1,
+    "armor": [ { "encumbrance": 2, "coverage": 100, "covers": [ "eyes" ] } ],
+    "flags": [ "panoramic_mask_outsert", "CANT_WEAR", "SUN_GLASSES", "FLASH_PROTECTION", "OUTER" ]
+  },
+  {
+    "id": "outsert_clear",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str": "clear outSert lens", "str_pl": "clear outSert lenses" },
+    "description": "A simple and flexible transparent lens of plastic that clips into place over a panoramic gas mask’s solitary eyepiece, providing extra protection from scratches, abrasions, and ballistic threats.",
+    "weight": "136 g",
+    "volume": "200 ml",
+    "price": 7300,
+    "price_postapoc": 200,
+    "material": [ "thermo_resin" ],
+    "symbol": "[",
+    "color": "light_gray",
+    "material_thickness": 1.5,
+    "armor": [ { "coverage": 100, "covers": [ "eyes" ], "rigid_layer_only": true } ],
+    "flags": [ "panoramic_mask_outsert", "CANT_WEAR", "OUTER" ]
+  },
+  {
+    "id": "outsert_breach",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str": "breach outSert lens", "str_pl": "breach outSert lenses" },
+    "description": "A specialized lens designed to clip into place over the solitary eyepiece of a panoramic gas mask, providing eye protection to users deploying a cutting torch while also improving impact resistance.",
+    "weight": "136 g",
+    "volume": "200 ml",
+    "price": 17034,
+    "price_postapoc": 200,
+    "material": [ "thermo_resin" ],
+    "symbol": "[",
+    "color": "light_gray",
+    "material_thickness": 1.5,
+    "armor": [ { "encumbrance": 35, "coverage": 100, "covers": [ "eyes" ] } ],
+    "qualities": [ [ "GLARE", 1 ] ],
+    "flags": [ "panoramic_mask_outsert", "SUN_GLASSES", "FLASH_PROTECTION", "CANT_WEAR", "OUTER" ]
+  },
+  {
+    "id": "insert_vision_correction",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str": "vision correction assembly", "str_pl": "vision correction assemblies" },
+    "description": "a detachable frame of prescription lenses intended for installation behind a panoramic gas mask’s visor, providing an alternative vision-correcting solution to users with inherent eye issues rather than contact lenses.",
```

Comment:
```suggestion
    "description": "A detachable frame of prescription lenses intended for installation behind a panoramic gas mask’s visor, providing an alternative vision-correcting solution to users with inherent eye issues rather than contact lenses.",
```